specifyweb/frontend/js_src/lib/components/Attachments/RecordSetAttachment.tsx

```diff
@@ -108,6 +108,9 @@ export function RecordSetAttachments<SCHEMA extends AnySchema>({
           className={{
             container: dialogClassNames.wideContainer,
           }}
+          dimensionsKey={
+            fetchedCount.current === records.length ? undefined : false
```

Comment:
what is `fetchedCount.current === records.length` supposed to represent
all attachments loaded?
some attachments loaded?

edit: looks like it means "isComplete". could you please extract this into a variable called `isComplete` and use it here, and also on line 144?
that would improve code clarity